cmd/all.go

```diff
@@ -10,6 +10,7 @@ import (
 
 func init() {
 	rootCmd.AddCommand(allCmd)
+	allCmd.Flags().BoolVarP(&interruptedUrls, "ignore", "i", false, "Ignore interrupted urls.")
```

Comment:
We can rename:
`ignore` with `restart`
`interruptedUrls` with `restartCrawling`
`Ignore interrupted urls.` with `Ignore interrupted jobs and restart from the beginning`
?

Reply:
Of course.